examples/external_inputs.rs

```diff
@@ -74,6 +74,8 @@ where
     F: Absorb,
 {
     type Params = PoseidonConfig<F>;
+    type ExternalInputs = VecF<F>;
+    type ExternalInputsVar = VecFpVar<F>;
```

Comment:
I might be too nitpicky, but do you think it is a good idea to replace `VecF<F>` and `VecFpVar<F>` with `[F; 1]` and `[FpVar<F>; 1]`, so we can remove `VecF` and `VecFpVar` defined in this file (not in `experimental-frontends/src/utils.rs`) and make the example simpler?

Although `[FpVar<F>; 1]` does not satisfy `Default`, we can in fact remove the `Default` trait bound from
https://github.com/privacy-scaling-explorations/sonobe/blob/0dde06067885b394f7cb9b291df6c5a5d72fd130/folding-schemes/src/frontend/mod.rs#L20
because we can call `ExternalInputsVar::new_witness(cs.clone(), || Ok(external_inputs))?` instead of `ExternalInputsVar::default()` when we need dummy external inputs.

Reply:
Absolutely! This makes a lot of sense and it's more clean, updated it: https://github.com/privacy-scaling-explorations/sonobe/pull/191/commits/7682fe0fa190fa3a229eb5e5d5e7d8e35e034734
Thank you very much!